iron-media-query.html

```diff
@@ -58,25 +58,26 @@
         type: Boolean,
         value: false
       },
-      
+
       /**
        * @type {function(MediaQueryList)}
-       */ 
+       */
       _boundMQHandler: {
         value: function() {
           return this.queryHandler.bind(this);
         }
       },
-      
+
       /**
        * @type {MediaQueryList}
-       */ 
+       */
       _mq: {
         value: null
       }
     },
 
     attached: function() {
+      this.style.display = 'none';
```

Comment:
And by slower I mean more expensive, but mobile github doesn't let me edit messages